lib/workflows.js

```diff
@@ -0,0 +1,21 @@
+/**
+ *  @fileoverview User functions index
+ *
+ *  @author  Scott Nath
+ *
+ */
+'use strict';
+
+const loader = require('./workflows/load');
+const model = require('./workflows/data');
```

Comment:
Why are you changing names around here, when it's so simple to have them be the same